why_choose_vinta.md

```diff
@@ -0,0 +1,13 @@
+Bill Gates once mentioned that: **“Your most unhappy customers are your greatest source of learning.”** Sometimes that points you to focusing more on the client’s needs, other times it warns you that you might be looking at the wrong customer. Here we describe the things Vinta looks for as the right customer type. Clients like these are the ones Vinta can deliver most value to. These aspects came from our experience in aligning business value with Vinta team’s values.
```

Comment:
What are your thoughts on having the title on this file:

#  Why Choose Vinta

Reply:
Makes sense. I just checked, we don't have a way to do it on the playbook (some of the newer pages have titles, most of the old ones, don't =/ ) I'll add an issue for that.